checkout/domain/placeorder/states/failed.go

```diff
@@ -4,33 +4,55 @@ import (
 	"context"
 
 	"flamingo.me/flamingo-commerce/v3/checkout/domain/placeorder/process"
+	"flamingo.me/flamingo/v3/framework/flamingo"
 	"go.opencensus.io/trace"
 )
 
 type (
 	// Failed state
 	Failed struct {
-		Reason process.FailedReason
+		Reason      process.FailedReason
```

Comment:
Don't see any usages of this struct field. Maybe we should get rid of it. The failed reason is stored in the overall process context anyways.